server/control.go

```diff
@@ -346,13 +346,15 @@ func (c Controller) runOnce(ctx context.Context, leaderKey string, tick <-chan t
 	newlyDrained := op.ChooseDrainedNodes(cluster, apiServers, rqEntries)
 	drainCompleted, drainTimedout, _ := op.CheckDrainCompletion(ctx, inf, nf.HealthyAPIServer(), cluster, rqEntries)
 	rebootDequeued := op.CheckRebootDequeue(ctx, cluster, rqEntries)
+	RebootCancelled := op.CheckRebootCancelled(ctx, cluster, rqEntries)
```

Comment:
Please use lowercase.

```suggestion
	rebootCancelled := op.CheckRebootCancelled(ctx, cluster, rqEntries)
```